src/proj7BittingCerratoCohenEllmer/bantam/lexer/Scanner.java

```diff
@@ -205,6 +228,14 @@ private boolean isCompleteMath(String spelling) {
         }
     }
 
+    private String lastSymbol(String spelling){
```

Comment:
These methods are a strong indication that we should be using a stack holding Characters instead of a String